src/register/Register.jsx

```diff
@@ -0,0 +1,272 @@
+import React, { useState } from "react";
+import "./register.css";
+
+
+export default function Register() {
+    // set user data state
+    const [validName, setValidName] = useState(true);
+    const [validUsername, setValidUsername] = useState(true);
+    const [validEmail, setValidEmail] = useState(true);
+    const [validPassword, setValidPassword] = useState(true);
+    const [terms_and_conditions_checked, setTermsAndConditionsChecked] = useState(false);
+    const [need_mentoring, setNeedMentoring] = useState(false);
+    const [available_to_mentor, setAvailableToMentor] = useState(false);
+    const [responseMessage, setResponseMessage] = useState(null);
+
+
+    const handleSubmit = async e => {
+        e.preventDefault();
+        console.log(e.target)
```

Comment:
No console logs in production code please

Reply:
removed all console.log() 👍